.travis.yml

```diff
@@ -18,9 +18,17 @@ before_install:
             sudo docker pull welder/web:latest
             sudo docker pull welder/web-with-coverage:latest
             sudo docker pull welder/web-e2e-tests:latest
-        else
+        fi
+install:
+  - |
+        if [ "$COMMAND" != "end-to-end-test" ]; then
             npm install
```

Comment:
This is not needed, the previous version in before_install was doing the same thing. Please revert.